Update camera timelapse video creation

Charlesworth opened the pull request:
Re issue #1917 update the time-lapse documentation to use `ffmpeg` to replace `libav-tools`, which is no longer install-able on Raspberry Pi OS:

```
Package libav-tools is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  ffmpeg

E: Package 'libav-tools' has no installation candidate
```

Tested on Raspberry Pi OS Lite Kernel version: 5.10

aallan commented:
LGTM. @JamesH65 ?

JamesH65 commented:
My only thought was that the glob stuff may need more explanation - most users will not understand the meaning.

lurch commented on usage/camera/raspicam/timelapse.md:
````diff
@@ -38,45 +40,30 @@ Make sure that you use e.g. `%04d` to make `raspistill` output each image to a n
-sudo apt install libav-tools
 ```
-Now you can use the tools to convert your JPEG files in to an H264 video file:
+sudo apt update
+sudo apt full-upgrade
+sudo apt install ffmpeg
````
@XECDesign Should we be recommending users to do a `sudo apt full-upgrade` before doing `sudo apt install ffmpeg` ?

Charlesworth replied:
I removed this bit. I initially tested with Raspberry Pi lite which required this, testing with a fresh copy of Raspberry Pi OS desktop, `ffmpeg` was actually already installed so update and upgrade was not required

lurch commented on usage/camera/raspicam/timelapse.md:
```diff
@@ -38,45 +40,30 @@ Make sure that you use e.g. `%04d` to make `raspistill` output each image to a n
+- `-f image2` Set ffmpeg to read from a list of image files specified by a pattern.
+- `-pattern_type glob` Select the pattern type used to interpret the provided filenames.
+- `-i 'image_*.jpg'` The input file specification (to match the files produced during the capture).
+- `-s 1280x720` Scale to 720p. You can also use 1920:1080, or lower resolutions, depending on your requirements. Note the Pi can only play back up to 1080p video, but if you are intending to play back at, for example, 4K, you could set that here.
```
I suspect the `1920:1080` ought to be `1920x1080` to match the syntax of `1280x720` ?
Also, I believe "the Pi can only play back up to 1080p video" is no longer true for the Pi4?

Charlesworth commented:
Thanks for the feedback everyone. I added some more explanation on `-pattern_type glob`, corrected the scaling flag notes and updated the installation instructions to remove the update and upgrade.

aallan commented:
Ready to merge @JamesH65 @lurch ?

JamesH65 commented:
LGTM.
